{"name": "test_vwo_login_negative", "status": "skipped", "statusDetails": {"message": "XFAIL reason: Failed\n\n_pytest.outcomes.XFailed: Failed", "trace": "setup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"7df30301e2a8b17a6a35f4f5baef3662\")>\n\n    @allure.epic(\"VWO Login Test\")\n    @allure.feature(\"TC#0 - VWO App Negative Test\")\n    @pytest.mark.negative\n    def test_vwo_login_negative(setup):\n        try:\n            driver = setup\n            loginPage = LoginPage(driver)\n            loginPage.login_to_vwo(usr=\"admin@admin@gmail.com\",pwd=\"admin\")\n            time.sleep(5)\n            error_message = loginPage.get_error_message_text()\n>           assert error_message == \"Yours email, password, IP address or location did not match\"\nE           AssertionError: assert 'Your email, ...did not match' == 'Yours email,...did not match'\nE             \nE             - Yours email, password, IP address or location did not match\nE             ?     -\nE             + Your email, password, IP address or location did not match\n\ntests/vwoLoginTests/test_vwlo_login.py:32: AssertionError\n\nDuring handling of the above exception, another exception occurred:\n\nsetup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"7df30301e2a8b17a6a35f4f5baef3662\")>\n\n    @allure.epic(\"VWO Login Test\")\n    @allure.feature(\"TC#0 - VWO App Negative Test\")\n    @pytest.mark.negative\n    def test_vwo_login_negative(setup):\n        try:\n            driver = setup\n            loginPage = LoginPage(driver)\n            loginPage.login_to_vwo(usr=\"admin@admin@gmail.com\",pwd=\"admin\")\n            time.sleep(5)\n            error_message = loginPage.get_error_message_text()\n            assert error_message == \"Yours email, password, IP address or location did not match\"\n        except Exception as e:\n>           pytest.xfail(\"Failed\")\nE           _pytest.outcomes.XFailed: Failed\n\ntests/vwoLoginTests/test_vwlo_login.py:34: XFailed"}, "start": 1715144396790, "stop": 1715144402280, "uuid": "c4b06b5b-9500-4bcc-9f80-ec70d5b15384", "historyId": "3fcd1855d8efcafbe5ebefeebcfd4d00", "testCaseId": "3fcd1855d8efcafbe5ebefeebcfd4d00", "fullName": "tests.vwoLoginTests.test_vwlo_login#test_vwo_login_negative", "labels": [{"name": "feature", "value": "TC#0 - VWO App Negative Test"}, {"name": "epic", "value": "VWO Login Test"}, {"name": "tag", "value": "negative"}, {"name": "parentSuite", "value": "tests.vwoLoginTests"}, {"name": "suite", "value": "test_vwlo_login"}, {"name": "host", "value": "MBP-C02DRR20MD6M"}, {"name": "thread", "value": "4567-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.vwoLoginTests.test_vwlo_login"}]}